.github/styles/Vocab/Docs/accept.txt

```diff
@@ -83,7 +83,7 @@ Postman
 [Pp]rivatelink
 psql
 Python
-Redis
+[Rr]edis
```

Comment:
Please remove this change, Redis is a product name and should always be written with its capital letter. It means that if you use it in lowercase, for example in a command, it should have code/literal markup so that the spell check doesn't trip over it.

Reply:
Oki